db/models.py

```diff
@@ -1,4 +1,12 @@
+from typing import Any
+
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
+
 from django.db import models
+from django.db.models import UniqueConstraint
+
+import settings
```

Comment:
https://docs.djangoproject.com/en/4.2/topics/settings/#using-settings-in-python-code